dist/aphrodite.js

```diff
@@ -1400,7 +1400,7 @@ function hyphenateStyleName(name) {
 }
 
 var hyphenateStyleName$1 = /*#__PURE__*/Object.freeze({
-  default: hyphenateStyleName
+  'default': hyphenateStyleName
```

Comment:
Please revert all the changes in dist, that will be updated as part of the publishing step.

Reply:
Done